src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -2393,46 +2352,72 @@ export function showWaterfallPlot(
     );
 }
 
+export function bothAxesNoMolecularProfile(horzSelection:AxisMenuSelection, vertSelection:AxisMenuSelection):boolean {
+    const noMolecularProfileDataTypes = [
+        GenericAssayTypeConstants.TREATMENT_RESPONSE,
+        CLIN_ATTR_DATA_TYPE,
+        NONE_SELECTED_OPTION_STRING_VALUE
+    ];
+    const horzDataType = horzSelection.dataType || '';
+    const vertDataType = vertSelection.dataType || '';
+    return (
+        noMolecularProfileDataTypes.includes(horzDataType) &&
+        noMolecularProfileDataTypes.includes(vertDataType));
+}
```

Comment:
I think we should give this function a better name to clarify both axes should be (TREATMENT_RESPONSE or CLIN_ATTR_DATA_TYPE or NONE_SELECTED_OPTION_STRING_VALUE), `bothAxesNoMolecularProfile` sounds like we just need both axes to have any dataType.

Reply:
@dippindots Do you have any specific suggestion to rename this function?

Reply:
@ngocnn1104 I just noticed we also have other similar names like `oneAxisMolecularProfile ` and `bothAxesMolecularProfile`. I think it's fine to keep this name. I also want to make sure `NoMolecularProfile` should include all `LIMIT_VALUE` data, right? (I need to change apply the logic to all `LIMIT_VALUE` profiles)

Reply:
@dippindots Thanks for clarifying this point for me via Slack. I'm resolving this conversation.